packages/aws-cdk/lib/commands/context.ts

```diff
@@ -1,5 +1,5 @@
 import * as chalk from 'chalk';
-import * as yargs from 'yargs';
+import type { Arguments } from 'yargs';
```

Comment:
You could reduce the changeset by literally doing

```suggestion
import type * as yargs from 'yargs';
```

The rest of the code then does not need changing.

Reply:
See https://github.com/aws/aws-cdk/pull/18667#discussion_r805444119